src/views/Home/components/FAQ/index.scss

```diff
@@ -0,0 +1,78 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.faq-component {
+  margin-top: -1px;
+  background-size: cover;
+  background: linear-gradient(
+    var(--color-light-pink),
+    var(--color-lavender),
+    var(--color-sky-blue)
+  );
+  &__box {
+    margin: auto;
+    border: 7px solid rgba(255, 255, 255, 0.75);
+    border-radius: 59px;
+    padding: 10px;
+    display: table;
+    // height: 1400px;
+    width: 80%;
+    @include responsiveness.screen(smallest, phone, desktop, big-desktop) {
+      width: 60%;
+    }
+  }
+  &__faqtitle {
+    font-family: "Source Sans Pro", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    @include responsiveness.font($responsive: 4.1vw, $min: 35px, $max: 60px);
+    text-align: center;
+    color: var(--color-text-primary);
+    margin-bottom: 10px;
+  }
+  &__line {
+    width: 25%;
+    height: 11px;
+    background: rgba(255, 255, 255, 0.73);
+    margin: auto;
+    // @include responsiveness.screen(desktop, big-desktop) {
+    //   width: 20px;
+    // }
+  }
+  &__question {
+    font-family: Lato, sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 24px;
+    line-height: auto;
+    color: #ffffff;
+    cursor: pointer;
+  }
+  &__answer {
+    font-family: Lato, sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: auto;
+    color: #ffffff;
+    display: none;
+  }
+
+  &__questionsandanswers:after {
+    display: table;
+    clear: both;
+  }
+  &__column {
+    vertical-align: middle;
+    float: left;
+    width: 33%;
+    @include responsiveness.screen(smallest, phone, tablet) {
+      width: 100%;
+    }
```

Comment:
<img width="328" alt="faqClippingLargeTablet" src="https://user-images.githubusercontent.com/38510322/140687135-a6e3c1db-42e6-40c7-ae75-0f880e3a5fad.PNG">

The responsiveness has improved immensely! There still some clippings ~800px width.